MODULES/rds/rds.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  db_name  = "ziyo"
+  username = "user_sam"
+}
+resource "aws_db_instance" "default" {
+  allocated_storage      = var.allocated_storage
+  db_name                = "${local.db_name}_${var.environment}"
+  engine                 = var.engine
+  engine_version         = var.engine_version
+  instance_class         = var.instance_class
+  username               = "${local.username}_${var.environment}"
+  password               = var.password
+  parameter_group_name   = "default.postgres12"
+  skip_final_snapshot    = var.skip_final_snapshot
+  db_subnet_group_name   = var.db_subnet_group_name
+  vpc_security_group_ids = var.vpc_security_group_ids
+  storage_type           = var.storage_type
+  tags = {
+    name        = "ziyotek"
+    environment = var.environment
+  }
+}
+
+resource "aws_db_subnet_group" "ziyo_subg" {
+  name       = "ziyo_sub_group"
```

Comment:
the name should also be environment specific ( include var.environment in the name)